CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 
 ## stream-chat-android-state
 ### 🐞 Fixed
+- Fixing first message for long threads. [#4620](https://github.com/GetStream/stream-chat-android/pull/4620)
```

Comment:
This hasn't been released and is not an issue in V5 so I don't think we need a changelog update.